src/components/product/OutOfStock/out-of-stock.scss

```diff
@@ -0,0 +1,51 @@
+@import "src/styles/scaffold";
+
+[data-store-out-of-stock] {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-top: var(--fs-spacing-4);
+
+  .text__title-subsection {
+    margin-bottom: var(--fs-spacing-0);
+  }
+}
+
+[data-store-out-of-stock-input] {
+  @include input-focus-ring;
+
+  width: 100%;
+  padding: var(--fs-spacing-1) var(--fs-spacing-7) var(--fs-spacing-1) var(--fs-spacing-2);
+  margin-top: var(--fs-spacing-3);
+  margin-bottom: var(--fs-spacing-3);
+  border: var(--fs-border-width-default) solid var(--fs-border-color-default);
+  border-radius: var(--fs-border-radius-default);
+  transition:
+    box-shadow var(--fs-transition-timing) var(--fs-transition-function),
+    border var(--fs-transition-timing) var(--fs-transition-function);
+
+  &:hover {
+    border-color: var(--fs-border-color-default-active);
+    box-shadow: 0 0 0 var(--fs-border-width-default) var(--fs-border-color-default-active);
+  }
+}
+
+[data-store-out-of-stock-button] {
+  display: inline-flex;
+  column-gap: var(--fs-spacing-2);
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  min-height: var(--fs-control-tap-size);
+  padding: var(--fs-spacing-1)  var(--fs-spacing-3);
+  font-weight: var(--fs-text-weight-bold);
+  text-decoration: none;
+  cursor: default;
+  border: 0;
```

Comment:
```suggestion
  width: 100%;
```

Reply:
We don't need to duplicate the `primary` button styles here, because it's already applied. We just need to fix the width, okay?  👍 

![image](https://user-images.githubusercontent.com/3356699/161093282-ffba84a2-a6b2-430a-81db-890947c0d050.png)

Reply:
Perfect, Fan!